README.md

```diff
@@ -1,124 +1,70 @@
-
-> **_[Note]_**  The advantage of this design is that the appropriate interface closely mimics the one of [WASI-NN](https://github.com/WebAssembly/wasi-nn) with its five functions `load()`, `init_execution_context()`, `set_input()`, 
-	`compute()` and `get_output()`. 
-
-
-<div style="width: 40%; height: 25%">
-
-![most basic design](images/most_basic_setup.png "Most basic design of an example application doing inference")
-</div>
-
-<div style="width: 40%; height: 25%">
-
-![legend for most basic design](images/legend_basic_design.png "Most basic design of an example application doing inference")
-
-</div>
 
-#### Data flow
+> **_[Information]_**  Other actions which are typical for the discipline of machine learning shall deliberately be excluded, e.g. training and data exploration.
 
-When a request comes in (1) via the http-server (http) it is routed (2) to the actor (M). The actor (M) registers (3) at the inference engine (IE) in case it did not do so before and provides the data (3) to the inference engine (IE). The result is passed (4) from the inference engine (IE) back to the actor (M) which, in turn, passes (5) the result to the http-server (https) where it is sent as a response.
+## Design decisions
 
-<div style="width: 40%; height: 25%">
+### Inference API actor
 
-![legend for most basic design](images/most_basic_design_data_flow.png "Most basic design of an example application doing inference")
+The Inference API actor (IAPI) shall
 
-</div>
+1. be linked to a HTTP server (http)
+2. expose a public API for requesting inference actions.
+3. validate its input
+4. make actor-to-actor calls to an inference actor (I)
+5. translate internal results to public HTTP output
 
-### More advanced design
+### Inference Actor
 
-A more advanced design additionally introduces an inference client (IC). The inference client (IC) communicates with the http-server (http) and the inference engine (IE) and may be regarded as a *reverse proxy* in that it deals with the opaque handles of the inference engine (IE) alias [GraphExecutionContext](https://github.com/Finfalter/wasmCloudArtefacts/blob/main/interfaces/mlinference/rust/src/mlinference.rs) internally but deals with (nick) names of AI models externally. 
+The Inference Actor (I) shall
 
-> **_[Note]_**  The use of an inference client (IC) allows an easy deployment of multiple actors (M). However, the interface of such a design would differ from an interface of the basic design.
+6. await actor-to-actor calls to perform actions
+7. perform any necessary validation on an incoming call
+8. request inference processing from the inference engine (IE) and return results
```

Comment:
I see these numbers are somewhat discontinuous, is that intended? It looks like these are separate lists, so I'd just suggest changing all of the numbers to `1.` and allowing Markdown to number automatically